arbitrum-docs/devs-how-tos/how-to-estimate-gas.mdx

```diff
@@ -0,0 +1,142 @@
+---
+id: how-to-estimate-gas
+title: "How to estimate gas in Arbitrum"
+description: Learn how to estimate gas before submitting transactions.
+author: TucksonDev
+---
+
+import PublicPreviewBannerPartial from '../partials/_public-preview-banner-partial.md'; 
+
+<PublicPreviewBannerPartial />
+
+This how-to is intended for users and developers who want to understand how gas is calculated in Arbitrum, and how it can be estimated before submitting transactions. More information about the theory behind these calculations can be found in this [Medium article](https://medium.com/offchainlabs/understanding-arbitrum-2-dimensional-fees-fd1d582596c9) and the [Gas and Fees](/arbos/gas) page. 
```

Comment:
no action required, just a future optimization - explicitly identifying a single target audience can help us cater to specific audience needs. Eg content for nontechnical users will differ significantly from content for devs (who have different prior knowledge / experience). 

We can try to normalize a pattern by which every document specifies a single `content-type` and `target-audience` to integrate this mechanism into our info architecture / content structure :).